det3d/datasets/nuscenes/nusc_common.py

```diff
@@ -400,12 +400,12 @@ def _fill_trainval_infos(nusc, train_scenes, val_scenes, test=False, nsweeps=10,
         ref_cam_path, _, ref_cam_intrinsic = nusc.get_sample_data(ref_cam_front_token)
 
         # Homogeneous transform from ego car frame to reference frame
-        ref_from_car = transform_matrix(
+        initial_car_SE3_reference = transform_matrix(
```

Comment:
thanks for the PR, Ayush. What does the reference frame represent?

I also prefer `egovehicle` to `car`, since car could represent another object, not the egovehicle

Reply:
Also didn't capture what `initial` represents -- is that t=0 before aggregation?

maybe `egot1_SE3_city`? (if the city is the reference?)

Reply:
Addressed in [this commit](https://github.com/hchittanuru3/CenterPoint/pull/3/commits/209cddf843a8962039acaffa9269c5a1a2ad58a7)